### DISCUSSION TITLE
Renaming of guns

### DISCUSSION THREAD

**Discussion by Lamandus** [CleverRaven/Cataclysm-DDA#45934] [General]
I wonder if we should consider to add the manufacture to every gun we have. Since we have for example the H&K MP5SD, but we call the Heckler & Koch MK 23: MK 23 MOD 0

what do you think about it?

**Comment by kevingranade** [comment]
There is no one answer for this, we use prevailing nomenclature, which might use the manufacturer name or might not.

**Reply by Lamandus** [reply]
so what would you prefer, personally? Right now it is a mix of both.

**Reply by kevingranade** [reply]
As I said, there is no one answer, which means it's a mix.

**Comment by actual-nh** [comment]
I'm not sure if it would be even workable to have a strict always-yes/always-no system. If the text gets too long, you need to leave off the manufacturer; on the other hand, if there would be a danger of confusion without the manufacturer, you need to keep the manufacturer's name.
